SygicUIKit/Classes/Views/ZoomControl/SYUIZoomController.swift

```diff
@@ -0,0 +1,123 @@
+public enum SYUIZoomActivity {
+    case zoomIn
+    case zoomOut
+    case startZoomIn
+    case startZoomOut
+    case stopZooming
+    case zoomingIn
+    case zoomingOut
+    case toggle3D
+}
+
+public protocol SYUIZoomControllerDelegate: class {
+    func zoomController(wants activity: SYUIZoomActivity)
+}
+
+public class SYUIZoomController: SYUIExpandableButtonsController {
+    
+    private let zoomInButton = SYUIExpandableButton(withType: .icon)
+    private let zoomOutButton = SYUIExpandableButton(withType: .icon)
+    private let toggle3DButton = SYUIExpandableButton(withType: .icon)
+    
+    public var is3D = false
```

Comment:
tu by si to komentar vyzadovalo kedy sa to meni a didSet by nemal nieco spravit?